tools/travis.sh

```diff
@@ -1,7 +1,10 @@
 set -x
 ulimit -n 4096
+# do this for our daily build test run
+if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+    powershell -c "Import-Module ./build.psm1; Start-PSBootstrap; Start-PSBuild -CrossGen; Start-PSPester -Tag @('CI','Feature','Scenario') -ExcludeTag RequireAdminOnWindows; Start-PSxUnit"
```

Comment:
> `Start-PSBootstrap`

Maybe we still want packages out of it? I.e. `Start-PSBootstrap -Package; ...; Start-PSPackage`